backup-restore/recipes/getResources.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/zsh
```

Comment:
Linux by default does not have zsh. Change to bash. That will also be consistent other scripts in the repo.

Reply:
Modified.